### PR TITLE
More complete tools

### PR DESCRIPTION
**Pull request by Alanamosse**

### CODE REVIEW

File: tools/gdal/gdal_translate.xml [hunk 1]
```diff
@@ -18,7 +55,73 @@
             <when value="simple">
             </when>
             <when value="advanced">
-				<param name="b" type="text" label="-b band" help="Select an input band band for output. Bands are numbered from 1. Multiple -b switches may be used to select a set of input bands to write to the output file, or to reorder bands." value="-b 1 -b 2 -b 3"/>
+            	<param name="strict" type="boolean" truevalue="-strict" falsevalue="" checked="false" label="-strict" help="Don't be forgiving of mismatches and lost data when translating to the output format."/>
```
Comment (bgruening) [inline tools/gdal/gdal_translate.xml]: The indentation should be 4 spaces instead of a tab.

File: tools/gdal/gdal_translate.xml [hunk 2]
```diff
@@ -18,7 +55,73 @@
...
+				<param name="r" type="select" label="-r" help="Select a resampling algorithm.">
+					<option value="nearest" selected="true"/>
+					<option value="bilinear"/>
+					<option value="cubic"/>
+					<option value="cubicspline"/>
+					<option value="lanczos"/>
+					<option value="average"/>
+					<option value="mode"/>
+				</param>	
+				<param name="outsize_x" type="text" label="-outsize xsize" value="ex : 75"/>
```
Comment (bgruening) [inline tools/gdal/gdal_translate.xml]: is this correct to be a text parameter? What does ex mean?

File: tools/gdal/gdal_translate.xml [hunk 3]
```diff
@@ -3,13 +3,50 @@
...
+				#end unless
+				
+				-r $settings.r
+				
+				#unless $settings.outsize_x=="ex : 75"				
+					-outsize xsize $settings.outsize_x
+				#end unless
+
+				#unless $settings.outsize_y=="ex : 75"				
+					-outsize ysize $settings.outsize_y
```
Comment (bgruening) [inline tools/gdal/gdal_translate.xml]: all text or data parameters should be single quoted `'`

File: tools/gdal/gdal_translate.xml [hunk 4]
```diff
@@ -18,7 +55,73 @@
...
+				
+				<conditional name="condi_scale">
+					<param name="scale" type="select" label="Rescale the input pixels values" help="Rescale the input pixels values from the range src_min to src_max to the range dst_min to dst_max. If omitted the output range is 0 to 255. If omitted the input range is automatically computed from the source data. Before GDAL 1.11, it can be specified only once, and in that case, it applies to all bands of the output dataset. Starting with GDAL 1.11, -scale can be repeated several times (if specified only once, it also applies to all bands of the output dataset), so as to specify per band parameters. It is also possible to use the '-scale_bn' syntax where bn is a band number (e.g. '-scale_2' for the 2nd band of the output dataset) to specify the parameters of one or several specific bands.">
+						<option value="no_scale" selected="true">Don't use the -scale option</option>
+						<option value="scale">Use the -scale option</option>
+					</param>
+					<when value="no_scale">
+					</when>
+					<when value="scale">
+						<param name="src_min" type="text" label="src_min" value="0"/>
```
Comment (bgruening) [inline tools/gdal/gdal_translate.xml]: integer?

File: tools/gdal/gdal_translate.xml [hunk 5]
```diff
@@ -18,7 +55,73 @@
...
+				
+				<conditional name="condi_srcwin">
+					<param name="srcwin" type="select" label="Selects a subwindow from the source image" help="Selects a subwindow from the source image for copying based on pixel/line location.">
+						<option value="no_srcwin" selected="true">Don't use the -srcwin option</option>
+						<option value="srcwin">Use the -srcwin option</option>
+					</param>
+					<when value="no_srcwin">
+					</when>
+					<when value="srcwin">
+						<param name="xoff" type="text" label="xoff" value="0"/>
```
Comment (bgruening) [inline tools/gdal/gdal_translate.xml]: integer?

File: tools/gdal/gdal_translate.xml (hunk 1 above)
Comment (Alanamosse) [reply tools/gdal/gdal_translate.xml]: Thks for the corrections !
Why should we prefer 4 spaces to a tab ?

Comment (bgruening) [reply tools/gdal/gdal_translate.xml]: It is just a best practice recommendation. No strict need, but we try all to stict to some standards to have a unified look and feel.

Also many more recommendations here: http://galaxy-iuc-standards.readthedocs.io/en/latest/index.html

Comment (Alanamosse) [reply tools/gdal/gdal_translate.xml]: Ok i'll try to force myself to use that standard, and i'll take a look at the iuc recommendations

**Pull request merged**